api/processes.json

```diff
@@ -1,7 +1,7 @@
 {
   "apps" : [{
     "name"               : "mockiji-api",
-    "script"             : "app.js",
+    "script"             : "app.js --config-file ../../mocks/data/dev.json",
```

Comment:
Yeah, I kinda hesitated before using that path, but that was the one referenced in `automation/bin/install` before.

I'm gonna move it back to the `automation/bin/install` and modify the README to include the `--api-base-path` argument so it uses the `api-simple-library` example.

Now concerning why I dropped the `env.json` file.
I may be wrong but I think the most logical way of using Mockiji would be to add it as a dev-dependency in a project and start it from a npm script (or gulp, webpack, ...):

```
{
  "name": "my-app",
  "scripts": {
    (...),
    "mocks": "mockiji --config-file ./mockiji.conf"
  },
  (...)
  "devDependencies": {
    "mockiji": "latest"
  }
}
```

 If that's that case, people are not going to edit an `env.json` file but rather reference to their own config file.